references/detection/transforms.py

```diff
@@ -437,3 +439,121 @@ def forward(
                 )
 
         return image, target
+
+
+def _copy_paste(image, target, paste_image, paste_target, inplace=True):
+
+    # Random paste targets selection:
+    num_masks = len(paste_target["masks"])
+    random_selection = torch.randint(0, num_masks, (num_masks,)).unique()
+
+    paste_masks = paste_target["masks"][random_selection]
+    paste_boxes = paste_target["boxes"][random_selection]
+    paste_labels = paste_target["labels"][random_selection]
+
+    masks = target["masks"]
+    # If source and paste data have different sizes
+    # Let's resize paste data
+    size1 = image.shape[-2:]
+    size2 = paste_image.shape[-2:]
+    if size1 != size2:
+        paste_image = F.resize(paste_image, size1, interpolation=F.InterpolationMode.BICUBIC)
+        paste_masks = F.resize(paste_masks, size1, interpolation=F.InterpolationMode.NEAREST)
+        # resize bboxes:
+        ratios = torch.tensor((size1[1] / size2[1], size1[0] / size2[0]), device=paste_boxes.device)
+        paste_boxes = paste_boxes.view(-1, 2, 2).mul(ratios).view(paste_boxes.shape)
+
+    paste_alpha_mask = paste_masks.sum(dim=0) > 0
+    paste_alpha_mask = F.gaussian_blur(paste_alpha_mask.unsqueeze(0), kernel_size=(5, 5), sigma=2.0)
+
+    # Copy-paste images:
+    image = (image * (~paste_alpha_mask)) + (paste_image * paste_alpha_mask)
```

Comment:
@vfdev-5 if `inplace` is `True`, I don't suppose this operation will make the changes on the original image right?

Reply:
Yes, you are right, we have to figure out if `inplace` arg can bring something helpful or we can simply drop it.

Reply:
I decided to drop the arg